docs/plasmapy_sphinx/automodsumm/generate.py

```diff
@@ -66,21 +66,10 @@ class AutomodsummRenderer(AutosummaryRenderer):
 
     app : `sphinx.application.Sphinx`
         Instance of the `sphinx` application.
-
-    template_dir : str
-        Path to a specified template directory.
     """
 
-    def __init__(
-        self,
-        app: Union["Builder", "Sphinx"],
-        template_dir: str = None,
-    ) -> None:
-
-        asumm_path = templates_dir
-        relpath = os.path.relpath(asumm_path, start=app.srcdir)
-        app.config.templates_path.append(relpath)
-        super().__init__(app, template_dir)
+    def __init__(self, app: "Sphinx") -> None:
+        super().__init__(app)
```

Comment:
The issue outlined in #1580 is the result of using `template_dir`, which had been deprecated since `v3.1` and officially removed in `v5.0`.

It was never doing anything, so removing it has no affect on the behavior of `plasmapy_sphinx`.